.github/workflows/daily-build.yml

```diff
@@ -31,22 +33,22 @@ jobs:
           echo -e '${{ toJson(secrets) }}' | jq -r 'to_entries[] | .key + "=" + .value' >> $GITHUB_ENV
 
       - name: Clone lib repo
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           repository: wso2-enterprise/wso2-sap-resources
           token: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          path: 'wso2-sap-resources'
+          path: 'libs'
 
       - name: Copy Libs to SAP Connector Directory
         run: |
-          cd $GITHUB_WORKSPACE
-          cp -r wso2-sap-resources/ballerina_resources/lib/. native/libs/
+          cp -r libs/ballerina_resources/lib/. jco/native/libs/
 
       - name: Build the Package
         env:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          cd jco
           ./gradlew build -x test -PbuildUsingDocker=nightly
 
       # Send notification when build fails
```

Comment:
Only notification is for daily build. Remove it from ci workflow

Reply:
Fixed with [12e315e](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/8/commits/12e315eb1c841256838179c834406987a7fbff59).